### PR TITLE
Yoochoose keys in env.php

### PR DESCRIPTION
**Pull request by kamilmusial**
@andrerom Please take a look.

### CODE REVIEW

**Comment by andrerom** [comment]
Ok, thought there would also be config.yml block, that is why I did not get why you wanted to skip it.

In this case you can omit changes to default_parameters.yml, and rather document that somewhere in case @jheba will need to inject it like that.

+1 with that cleaned up.

**Comment by kamilmusial** [comment]
I created an issue https://jira.ez.no/browse/EZS-818 and I can place it in some doc page if needed.

**Comment by andrerom** [comment]
+1

**Comment by lserwatka** [comment]
+1

File: app/config/env.php [hunk 1]
```diff
@@ -82,3 +82,20 @@
...
+// Yoochoose settings
+if ($value = getenv('YOOCHOOSE_CUSTOMER_ID')) {
+    $container->setParameter('ez_recommendation.default.yoochoose.customer_id', $value);
+}
+
+if ($value = getenv('YOOCHOOSE_LICENSE_KEY')) {
+    $container->setParameter('ez_recommendation.default.yoochoose.license_key', $value);
+}
+
+if ($value = getenv('YOOCHOOSE_SERVER_URI')) {
```
Comment (andrerom) [inline app/config/env.php]: In retrospective naming is somewhat misleading here as it's not really yochoose server uri, it's the one for eZ Publish install. ref: https://github.com/ezsystems/EzSystemsRecommendationBundle/blob/master/DependencyInjection/Configuration.php#L48

File: app/config/env.php [hunk 2]
```diff
@@ -82,3 +82,20 @@
...
+    $container->setParameter('ez_recommendation.default.server_uri', $value);
+}
+
+if ($value = getenv('YOOCHOOSE_INCLUDED_CONTENT_TYPES')) {
```
Comment (andrerom) [inline app/config/env.php]: We'll see tomorrow, but might be blocker here, as env variables will only be able to inject scalar values.
If Studio-demo had a sane default here, then that would have resolved the blocker (defining content type is more for the application to define anyway, not the environment).

Comment (kamilmusial) [inline app/config/env.php]: I was thinking about that, what about 
Dockerfile:
```
export YOOCHOOSE_INCLUDED_CONTENT_TYPES='article blog_post place'
```

env.php:
```
$container->setParameter('ez_recommendation.default.recommender.included_content_types', explode(' ', $value));
```
?
I didn't work with Docker, so I don't know if that's a good practice.

File: app/config/env.php (hunk 1 above)
Comment (kamilmusial) [inline app/config/env.php]: True, I wanted to group that with same prefixes.

File: app/config/env.php (hunk 2 above)
Comment (kamilmusial) [inline app/config/env.php]: Please take a look at [my commit](https://github.com/kamilmusial/ezstudio-demo/commit/ce581a492eb6700eac557a7b7fac45e421700ba2)

Comment (andrerom) [inline app/config/env.php]: Looks ok, but I'd prefer if we skip YOOCHOOSE_INCLUDED_CONTENT_TYPES, and instead you set a value that makes sense for studio demo in default_parameters.yml. As I said it's a setting more tied to what kind og demo and demo content you install, not to your subscription.

Comment (kamilmusial) [inline app/config/env.php]: @andrerom  [Fixed](https://github.com/kamilmusial/ezstudio-demo/commit/d0e877373f0d12b7d300cf2a5bac2da398dfb3a6) in my fork

Comment (andrerom) [inline app/config/env.php]: added some comments, with that +1

Comment (kamilmusial) [inline app/config/env.php]: Should I create a PR to master of 1.4 branch?

Comment (andrerom) [inline app/config/env.php]: 1.4 branch (if there is one)

Comment (lserwatka) [inline app/config/env.php]: I will create one.

**Pull request merged**